actions/access_keys/revoke.go

```diff
@@ -9,8 +9,16 @@ import (
 )
 
 // create will make a new model using the services defined in the action object
```

Comment:
Maybe the comment is wrong: "// create will make a new model u..."

Reply:
Yeah, you're right. This comment is wrong. Fixed